lib/symengine/real_double.rb

```diff
@@ -0,0 +1,7 @@
+module SymEngine
+  class RealDouble
+    def to_f
+        self.to_s.to_f
```

Comment:
Same here. You can do this in C API without having to convert it into string first.